lib/gemstash/migrations/02_authorizations.rb

```diff
@@ -2,7 +2,7 @@
   change do
     create_table :authorizations do
       primary_key :id
-      String :auth_key, :size => 2056, :null => false
+      String :auth_key, :size => 255, :null => false
```

Comment:
Docs make sense.

From the enoughness point of view, 32 chars (+4 dashes) is the length of a uuid. UUIDs are considered as universal unique ids. There is a really [good page on the wikipedia](https://en.wikipedia.org/wiki/Universally_unique_identifier#Random_UUID_probability_of_duplicates) that explains what is the probability of finding a duplicate (a thing that can actually happen) and it looks to be highly unlikely.

In other words, I wouldn't worry about running out of uuids any time soon, and we still have room for putting 7 uuids in a row as an auth key.